app/services/schools/milestone_swap_service.rb

```diff
@@ -0,0 +1,26 @@
+module Schools
+  class MilestoneSwapService
+    def initialize(milestones, target, direction)
+      @milestones = milestones
```

Comment:
I don't think. service will need to accept milestones; 
It will just need a target and direction.